aggregator/settings/base.py

```diff
@@ -103,15 +104,15 @@
 
 STATIC_URL = "/static/"
 
-from dc_theme.settings import (  # noqa
-    get_pipeline_settings,
-    STATICFILES_STORAGE,
-    STATICFILES_FINDERS,
-    SASS_INCLUDE_PATHS,
-)
+from dc_utils.settings.pipeline import *  # noqa
+from dc_utils.settings.pipeline import get_pipeline_settings
```

Comment:
I would expect 108 is redundant after 107

Reply:
FYI I removed this, but then flakes gave an error that I couldn't figure out how to get around. Added back in, I assume like I did last time I did this!